### PR TITLE
Nilesh | updated username for debian and rhel platform_family

### PR DESCRIPTION
**Pull request by Bairagi**
Updated secret path for debian and rhel family.

### CODE REVIEW

File: recipes/deploy_app.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  content deploy_key
+end
+
+template "#{node['deploy_app']['deploy_user_home']}/.ssh/config" do
+  source  "config.erb"
+  owner   node['deploy_app']['deploy_user']
+  group   node['deploy_app']['deploy_user']
+  mode    "0644"
+  variables({
+    :repo_host => "github.com",
```
Comment (vipulnsward) [inline recipes/deploy_app.rb]: Please use ruby 1.9 hash syntax.

File: recipes/deploy_app.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    :keyfile => "#{node['deploy_app']['deploy_user_home']}/.ssh/deploy"
```
Comment (vipulnsward) [inline recipes/deploy_app.rb]: same here and other places

**Comment by Bairagi** [comment]
@vipulnsward Updated PR.

File: recipes/deploy_app.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Cookbook Name:: boxy-rails
+# Recipe:: deploy_app
+
+secret = Chef::EncryptedDataBagItem.load_secret("#{node['boxy-rails']['secret']}")
```
Comment (vipulnsward) [inline recipes/deploy_app.rb]: Please fix this. It should be `secret_key_path` now i guess?

File: recipes/deploy_app.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Cookbook Name:: boxy-rails
```
Comment (vipulnsward) [inline recipes/deploy_app.rb]: If this recipe is not in use. Please delete it.

Comment (vipulnsward) [inline recipes/deploy_app.rb]: @Bairagi any issues with this? If not, is this ready to be merged?

**Comment by Bairagi** [comment]
@vipulnsward we can merge this.

**Pull request merged**